packages/slate/src/interfaces/editor.ts

```diff
@@ -1241,275 +1241,6 @@ export const Editor = {
     return text
   },
 
-  /**
```

Comment:
the removed snippet is the only reference to `immer`, so you can remove the imports on the second line

Reply:
good catch, thanks!